common/src/main/java/com/wynntils/mc/mixin/ScreenMixin.java

```diff
@@ -105,7 +105,7 @@ private void renderTooltipPost(PoseStack poseStack, ItemStack itemStack, int mou
     }
 
     @Inject(
-            method = "init(Lnet/minecraft/client/Minecraft;II)V",
+            method = "rebuildWidgets",
```

Comment:
Have they renamed the top-level "init" method? :-o If so, then the WynntilsScreenWrapper would fail..?

Or maybe they only extracted the part that called our init method into a separate method?

Reply:
No, it is worse that that :). We have `init(Minecraft minecraft, int width, int height)`, does the same as the old one, except it does not call another init. It calls `rebuildWidgets`, which in turn calls the secondary init.

```java
public void init(Minecraft minecraft, int width, int height) {
        this.minecraft = minecraft;
        this.itemRenderer = minecraft.getItemRenderer();
        this.font = minecraft.font;
        this.width = width;
        this.height = height;
        this.rebuildWidgets();
        this.triggerImmediateNarration(false);
        this.suppressNarration(NARRATE_SUPPRESS_AFTER_INIT_TIME);
    }
```

```java
protected void rebuildWidgets() {
        this.clearWidgets();
        this.setFocused((GuiEventListener)null);
        this.init();
    }
```

```java
protected void init() {
    }
```

Reply:
ok, that was what i meant by "they only extracted the part that called our init method into a separate method". But that's fine. This is the correct fix for that.